doc/rules/whitespace/spaces_inside_parenthesis.rst

```diff
@@ -0,0 +1,82 @@
+==================================
+Rule ``spaces_inside_parenthesis``
+==================================
+
+Parenthesis must be declared using the configured syntax.
+
+Configuration
+-------------
+
+``space``
+~~~~~~~~~
+
+whether to have ``spaces`` or ``none`` spaces inside parenthesis.
```

Comment:
is any community using `none` ?

Reply:
@keradus setting it to `none` which is the default is also equivalent to setting `no_spaces_inside_parenthesis => true` in config

Reply:
yes, you are right.
let me set the question properly:
is any community using `spaces`? i believe it's against PSR

Reply:
@keradus as mentioned in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4706 the WordPress community is using it